.github/workflows/build.yml

```
@@ -1,5 +1,18 @@
 name: Build
 
+env:
+  include_paths: |
+              charsets/
+              docs/
+              extra/
+              layouts/
+              masks/
+              modules/
+              OpenCL/
+              rules/
+              example*
+              hashcat*
```

Do not exclude `libhashcat.so.*` files.

```suggestion
              hashcat*
              libhashcat*
```